python/llm/setup.py

```diff
@@ -307,7 +319,9 @@ def setup_package():
             ]
         },
         extras_require={"all": all_requires,
-                        "xpu": xpu_requires,
+                        "xpu": xpu_requires,  # default to ipex 2.0 for linux and 2.1 for windows
+                        "xpu_20": xpu_20_requires,
+                        "xpu_21": xpu_21_requires,
```

Comment:
Could we install win with ipex 2.1 with `xpu_21` option?

Reply:
Yes, now on win, `xpu` and `xpu_21` will install ipex 2.1